### PR TITLE
add 'vc-export' module to export Verifiable Credentials

### PR DESCRIPTION
**Pull request by amarts**
vc-export module provides methods to convert `IDocument` (the verifiable
documents created from '@cord.network/sdk') type of json to `VerifiableCredential`
type of document (ie, the w3c VC standard compliant json). This method also
provides methods to prepare VP (Verifiable Presentation) from the IDocument,
and all methods to verify the proofs attached.

### CODE REVIEW

File: packages/modules/src/document/Document.ts [hunk 1]
```diff
@@ -71,12 +71,15 @@ function getHashLeaves(
...
 export function calculateDocumentHash(document: Partial<IDocument>): Hash {
+  /*
   const hashes = getHashLeaves(
     document.contentHashes || [],
     document.evidenceIds || [],
     document.createdAt || '',
     document.validUntil || ''
   )
+  */
+  const hashes = getHashLeaves(document.contentHashes || [], [], '', '');
```
Comment (amarts) [inline packages/modules/src/document/Document.ts]: This needs to be properly discussed.

File: packages/modules/src/content/Content.ts [hunk 2]
```diff
@@ -34,8 +34,8 @@ function jsonLDcontents(
   if (!schemaId) new SDKErrors.SchemaIdentifierMissingError()
   const vocabulary = `${schemaId}#`
   const result: Record<string, unknown> = {}
-  if (issuer) result['@issuer'] = issuer
-  if (holder) result['@holder'] = holder
+  if (issuer) result['issuer'] = issuer
+  if (holder) result['holder'] = holder
```
Comment (amarts) [inline packages/modules/src/content/Content.ts:38]: discuss on value of `@holder` -> holder change.

File: packages/vc-export/README.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+const result = await vcjs.verifyCredential({
+  credential,
+  suite: [signatureSuite, integritySuite, attestedSuite],
+  purpose,
+  documentLoader,
+})
+
+// 5. make sure all `results` indicate successful verification
+const verified = result.results.every((i) => i.verified === true)
+```
```
Comment (amarts) [inline packages/vc-export/README.md:108]: This document needs a proper updation.

File: packages/vc-export/package.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@cord.network/vc-export",
+  "version": "0.8.0-beta.8",
```
Comment (amarts) [inline packages/vc-export/package.json]: needs version to match with other package versions.

File: packages/vc-export/src/constants.ts [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+export const CORD_VERIFIABLE_CREDENTIAL_TYPE = 'CordCredential2020'
+
+export const CORD_STREAM_SIGNATURE_PROOF_TYPE = 'CordStreamSignature2020'
+export const CORD_SELF_SIGNATURE_PROOF_TYPE = 'CordSelfSignature2020'
+export const CORD_ANCHORED_PROOF_TYPE = 'CordCredential2020'
+export const CORD_CREDENTIAL_DIGEST_PROOF_TYPE = 'CordCredentialDigest2020'
+
+export const JSON_SCHEMA_TYPE = 'JsonSchemaValidator2018'
+
+export const CORD_CREDENTIAL_IRI_PREFIX = 'cred:cord:'
```
Comment (amarts) [inline packages/vc-export/src/constants.ts:28]: this is ok? do we need something else?

**Comment by amarts** [comment]
Output at the end of `demo-vc` now.

✅ Stream-Signature-Proof true ✧ Stream-Proof true ✧ Digest-Proof true ✧ Self-Signature-Proof true

✅ Credential revoked!
❌ Stream-Signature-Proof true ✧ Stream-Proof {verified: false, status: 'revoked'} ✧ Digest-Proof true ✧ Self-Signature-Proof true

**Review by smohan-dw — APPROVED** [review]
LGTM

**Pull request merged**